modules/cluster-logging-release-notes-5.0.9.adoc

```diff
@@ -0,0 +1,25 @@
+[id="cluster-logging-release-notes-5-0-9"]
+= OpenShift Logging 5.0.9
+
+This release includes link:https://access.redhat.com/errata/RHBA-2021:3705[RHBA-2021:3705 - Bug Fix Advisory. OpenShift Logging Bug Fix Release (5.0.9)].
+
+[id="openshift-logging-5-0-9-bug-fixes"]
+== Bug fixes
+
+This release also includes the following bug fixes:
+
+* Before this update, some log entries had unrecognized UTF-8 bytes, which caused Elasticsearch to reject the messages and block the entire buffered payload. This update resolves the issue: rejected payloads drop the invalid log entries and resubmit the remaining entries. (link:https://issues.redhat.com/browse/LOG-1574[LOG-1574])
```

Comment:
which messages? If that's unspecified, remove "the" in front of messages?

Reply:
Updated. Thanks.